agent/tcs/handler/types.go

```diff
@@ -37,7 +37,6 @@ type TelemetrySessionParams struct {
 	CredentialProvider            *credentials.Credentials
 	Cfg                           *config.Config
 	DeregisterInstanceEventStream *eventstream.EventStream
-	AcceptInvalidCert             bool
```

Comment:
Removed because this parameter is not used. `Cfg.AcceptInsecureCert` is used instead.